projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_base.py

```diff
@@ -703,3 +717,12 @@ def __verify_payload_format(self, payload_dict: dict):
                     resolvable_by=ResolvableBy.USER_ERROR,
                 )
             )
+
+    @staticmethod
+    def __object_is_data_frame(obj):
```

Comment:
Please, please use type hints 

```suggestion
    def __object_is_data_frame(obj: Union[dict,"DataFrame"]) -> bool:
```